PaddleCV/PaddleDetection/docs/GETTING_STARTED.md

```diff
@@ -75,8 +75,13 @@ path, simply add a `--save_file=` flag.
 
 ## FAQ
 
+**Q**: Why do I get `NaN` loss values during single GPU training?
+**A**: The default learning rate is tuned to multi-GPU training (8x GPUs), it must
+be adapted for single GPU training accordingly (e.g., divide by 8).
 
-Q: Why do I get `NaN` loss values during single GPU training?
 
-A: The default learning rate is tuned to multi-GPU training (8x GPUs), it must
-be adapted for single GPU training accordingly (e.g., divide by 8).
+**Q**: How to further reduce GPU memory?
```

Comment:
**Q**: How to reduce GPU memory usage?
**A**: Set environment variable `FLAGS_conv_workspace_size_limit` to a smaller number can reduce GPU memory footprint without affecting training speed.
Take Mask-RCNN (R50) as example, by setting `export FLAGS_conv_workspace_size_limit=512`, batch size could reach 4 per GPU.

Reply:
Done. Thanks!